### DISCUSSION TITLE
Private phone someone downloaded software to

### DISCUSSION THREAD

**Discussion by Breaktheglass12929** [community/community#66392] [Pull Requests]
### Select Topic Area

Question

### Body

Hi I have a private phone and and I think someone downloaded open source software from here onto it is this possible ?

**Comment by Akash1134** [comment]
Hi @Breaktheglass12929 ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
